src/fc/fc_domain.c

```diff
@@ -354,6 +354,45 @@ fcStartVMProcess(virFCDriver *driver G_GNUC_UNUSED,
     return ret;
 }
 
+static int
+fcConfigNetworkInterfaceVM(virFCDriver *driver G_GNUC_UNUSED,
```

Comment:
the name is pretty verbose. what about `fcConfigNetworkInterface`

Reply:
Sadly no, it has a check for not declaring variables in a loop